flocker/control/_clusterstate.py

```diff
@@ -58,3 +69,10 @@ def apply_changes(self, changes):
             self._deployment_state = change.update_cluster_state(
                 self._deployment_state
             )
+        # for change in changes:
+        #     wiper = change.get_information_wipe()
+        #     key = wiper.key()
+        #     self._information_wipers = self._information_wipers.set(
+        #         key, wiper)
+        #     self._information_wipers_added = self._information_wipers.set(
+        #         key, time())
```

Comment:
If you feel like it, `_information_wipers_added` could probably be a pvector of (timestamp, key) that gets appended to.  The cleanup step can just process a prefix of it, then, instead of visiting the entire collection.  If not, file a follow-up ticket.